src/components/shortcuts/OverallComparer.ts

```diff
@@ -5,21 +5,36 @@
  * Other licensing options may be available, please reach out to data-viz@feedzai.com for more information.
  */
 
-import { median } from "../../utils/maths";
-import { getOrdinalNumber } from "../../utils/maths";
+import { median, getOrdinalNumber } from "../../utils/maths";
 
-//makes the comparison between the focusedData and the overall data
-export function overallComparer(event, alertDiv, insights, arrayConverted, focusedData) {
-	const { nativeEvent } = event;
+/**
+ * Produces a message based on corner cases and calls comparer otherwise.
```

Comment:
What exactly do you mean about the "message based on corner cases"? Is that the focus of the function or is this secondary?